bin/config.py

```diff
@@ -0,0 +1,6 @@
+"""Configuration file for server.py"""
+import os
+
+PORT = os.environ.get('PORT')
```

Comment:
Add default param to `get` as here: https://github.com/tolstoyevsky/cusdeb-api/blob/master/cdapi/docker.py#L12.
Alse for `MEDIAWIKI_URL` and `CLIENT_URL`.